system-int-test/test/cmr/system_int_test/search/acls/variable_association.clj

```diff
@@ -8,6 +8,7 @@
    [cmr.system-int-test.data2.collection :as dc]
    [cmr.system-int-test.data2.core :as d]
    [cmr.system-int-test.system :as s]
+   [cmr.system-int-test.utils.association-util :as au]
```

Comment:
I think we prefer longer aliases now (I would name this association-util).

Reply:
It is used in too many places where the lines are too long and with a long alias the code becomes less readable. This is a conscious decision to use a shorter one here similar to what we did in the line above it.